spec/requests/api/v1/keywords_controller_spec.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe(Api::V1::KeywordsController) do
+  describe 'GET #index' do
+    context 'given one page of keywords' do
+      it 'returns all keywords' do
```

Comment:
i think we do not need to test just, just test the `it 'returns the first page of keywords' do` is enough

Reply:
Does this mean reduces these test cases, from:
```
context 'given one page of keywords' do
  it 'returns all keywords' do
  end
end

context 'given more than one page of keywords' do
  it 'returns the first page of keywords' do
  end
end
```

to only:
```
it 'returns the first page of keywords' do
end
```

Reply:
Yeah @Goose97, what do you think?

Reply:
I think it's fine to remove those tests. Fixed in 158a1be6c9b3341f7f14c9e57387f4df4ffef04c